### ISSUE TITLE
npm package name?

### ISSUE THREAD

**Issue by leegee** [isaacs/node-glob#514]
The docs for this repo say to "`npm i glob`", but when I do I seem to get a different distribution to when I do `npm i node-glob`.

Is it me?

**Comment by isaacs** [comment]
The `node-glob` package on npm isn't this module, it's a different thing entirely, which has not been updated in the last 7 years. 

What version of glob are you installing? Run `npm ls glob` and it'll tell you.

**Comment by isaacs** [comment]
Yeah, this module predates when I'd just always make the git repo name match the npm package name.  I should maybe rename it to `isaacs/glob`, but that's always so disruptive.

**Comment by Zamiell** [comment]
I don't think a repository rename is that disruptive, is it? Going forward, it would be less confusing for users if the repository name matched the npm name.

**Comment by frontzhm** [comment]
我看邮件的反应很迟钝。。。如果有事发短信提醒我看邮件。。。。。。

**Comment by isaacs** [comment]
@Zamiell it's still potentially pretty disruptive, if something is as old and has as many forks as this repo. It dates to a convention of naming repositories `node-<pkg name>`, which was very common back when this was created.
